geopandas/geoseries.py

```diff
@@ -288,6 +288,33 @@ def y(self):
         """
         return _delegate_property("y", self)
 
+    @property
+    def z(self):
+        """Return the z location of point geometries in a GeoSeries
+
+        Returns
+        -------
+        pandas.Series
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import Point
+        >>> s = geopandas.GeoSeries([Point(1, 1), Point(2, 2), Point(3, 3)])
```

Comment:
```suggestion
        >>> s = geopandas.GeoSeries([Point(1, 1, 1), Point(2, 2, 2), Point(3, 3, 3)])
```

We need Z geometry to work with z coordinates :).

Reply:
Sure:)